packages/transformers/pwa-manifest/src/PWAManifestTransformer.js

```diff
@@ -0,0 +1,57 @@
+// @flow
+// https://developer.mozilla.org/en-US/docs/Web/Manifest
+
+import {Transformer} from '@parcel/plugin';
+import type {MutableAsset} from '@parcel/types';
+
+import nullthrows from 'nullthrows';
+
+const getSrcHandler = (opt = {}) => (asset, dep) => {
+  dep.src = asset.addURLDependency(dep.src, opt);
```

Comment:
I add this for urls rewriting and get following from [this manifest](https://github.com/Drapegnik/parcel/blob/transformer-webmanifest/packages/core/integration-tests/test/integration/pwa-manifest/manifest.webmanifest):

```json
{
  "name": "pwa-webmanifest-example",
  "icons": [
    {
      "src": "1eda49598769bd3a3cd2b1e31d3cfaff",
      "sizes": "100x100",
      "type": "image/png"
    }
  ],
  "screenshots": [
    {
      "src": "f432622196df181e7734bdfd95fa62d0",
      "sizes": "100x100",
      "type": "image/png"
    },
    { "src": "" }
  ],
  "serviceworker": { "src": "d771eacd0fc5e521bf134585524d4727" }
}
```

Reply:
Yep, those ids will be replaced later with urls once the final bundle names are known.

Reply:
How can I test it?
```bash
yarn build
parceldev build --no-cache packages/core/integration-tests/test/integration/pwa-manifest/inde
x.html
```

produces the same file (with ids)

Reply:
@devongovett @DeMoorJasper @padmaia @wbinnssmith,

Could you tell me where is this final replacement (ids -> urls) occurs?
Looks like `.webmanifest` is ignored.

Reply:
never mind, I found - `generateDepToBundlePath`)

Reply:
the problem was because `.webmanifest` was being processed by `RawPackager`, which returns stream, so `typeof contents !== 'string'` and `generateDepToBundlePath` was skipped

I fixed this by adding `StringPackager` - not sure if this is the best way